specs/_features/eip7805/p2p-interface.md

```diff
@@ -0,0 +1,90 @@
+# EIP-7805 -- Networking
+
+This document contains the consensus-layer networking specification for EIP-7805.
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Time parameters](#time-parameters)
+- [Configuration](#configuration)
+- [The gossip domain: gossipsub](#the-gossip-domain-gossipsub)
+  - [Topics and messages](#topics-and-messages)
+    - [Global topics](#global-topics)
+      - [`inclusion_list`](#inclusion_list)
+- [The Req/Resp domain](#the-reqresp-domain)
+  - [Messages](#messages)
+    - [InclusionListByCommitteeIndices v1](#inclusionlistbycommitteeindices-v1)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+### Time parameters
+
+| Name | Value | Unit | Duration |
+| - | - | :-: | :-: |
+| `ATTESTATION_DEADLINE` | `uint64(4)` | seconds | 4 seconds |
```

Comment:
should time parameters be defined as a fraction of `SECONDS_PER_SLOT` instead, or do we just define them as part of presets to work with minimal as well?

Reply:
Oh you're right, this wouldn't work with minimal... I'll look into the best way to handle this later.

Reply:
Yes, that would work here. But not for other time values like:

https://github.com/ethereum/consensus-specs/blob/278145999fa7d9708a9872d5b11443f57014d108/specs/_features/eip7805/validator.md?plain=1#L45

Maybe that could be `SECONDS_PER_SLOT - 1` instead.

Reply:
Also, these time values are considered constants & `SECONDS_PER_SLOT` is configuration value. So it's not really possible to define a constant with a non-constant. If we do it like this, the times will need to be configs too.

Reply:
Thanks @bleesherman. I've done just that. Please see: https://github.com/ethereum/consensus-specs/pull/4003/commits/7a2cb7a1c3564d6dcc6ba4f64be8d66824336876

Reply:
Thinking about this more, not sure if having those values defined in preset will get us where we want as during testing we might use mainnet preset and just reduce seconds per slot. This would also make reducing seconds per slot in the future harder as you need to update a bunch of time parameters. But this might be something to worry about later

Reply:
To be clear, these are configuration values now. As of that commit I shared.

Reply:
Might be specific to how we handle this in Lodestar but even if it's part of config we can't derive the value from `SECONDS_PER_SLOT`

Reply:
Ah dang. Well I suppose clients could use the values without deriving them.

https://github.com/ethereum/consensus-specs/blob/7a2cb7a1c3564d6dcc6ba4f64be8d66824336876/configs/mainnet.yaml#L187-L189

https://github.com/ethereum/consensus-specs/blob/7a2cb7a1c3564d6dcc6ba4f64be8d66824336876/configs/minimal.yaml#L186-L188

Reply:
> Well I suppose clients could use the values without deriving them.

That's what I did for now but will break if we change seconds per slot with mainnet config but it's possible to adapt these values as well now since we have them in config

Reply:
@jtraglia @terencechain I think we might wanna clarify how to handle these timing parameters before merging, I don't think it works to have them as part of config or at least it causes a bunch of issues, for example, if you change `SECONDS_PER_SLOT` on kurtosis then these timing parameters also need to be adapted manually. I think it would be best to just have them documented as a fractions of seconds per slot which is already the case but not add them in config but rather inlined in the function call.

An example of this where the spec already uses this approach is in fork choice [on_block](https://github.com/ethereum/consensus-specs/blob/dev/specs/deneb/fork-choice.md#on_block)

```python
is_before_attesting_interval = time_into_slot < SECONDS_PER_SLOT // INTERVALS_PER_SLOT
```